benches/bench.rs

```diff
@@ -23,7 +23,7 @@ fn slow_sample_time() -> Duration {
 }
 
 fn sample_time() -> Duration {
-	std::env::var("SAMPLE_TIME").map_or(Duration::from_secs(10), |val| {
```

Comment:
Dumb question; what does this effect/do?

Reply:
Ooh I see; it's how long it'll repeat the benchmark for to gather data.

Reply:
is measurement time better?

Reply:
that'd be my choice personally :)